rdt/hyper_transformer.py

```diff
@@ -73,6 +75,42 @@ class HyperTransformer:
         'b': 'boolean',
         'M': 'datetime',
     }
+    DEFAULT_TRANSFORMERS = {
+        'numerical': NumericalTransformer,
+        'integer': NumericalTransformer(dtype=int),
```

Comment:
Maybe we should create specific transformers for each configuration, so this dictionary can contain classes instead of instances? I'm opening a new issue to discuss this there.